File: buzzvil-sdk-ios/buzzvil-sdk-ios-objc/Feed/FeedContainerViewController.m
@import BuzzvilSDK;
#import "FeedContainerViewController.h"

@interface FeedContainerViewController ()

@property (nonatomic, strong, readonly) BZVBuzzAdFeed *buzzAdFeed;

@end

@implementation FeedContainerViewController

- (void)viewDidLoad {
  [super viewDidLoad];

  _benefitHub = [BZVBuzzAdFeed feedWithBlock:^(BZVBenefitHubBuilder * _Nonnull builder) {}];
  [self displayContentViewController:[_buzzAdFeed viewController]];
}

- (void)displayContentViewController:(UIViewController*)contentViewController {
  [self addChildViewController:contentViewController];
  contentViewController.view.frame = self.view.bounds;
  [self.view addSubview:contentViewController.view];
  [contentViewController didMoveToParentViewController:self];
}

@end

File: buzzvil-sdk-v5-ios/buzzvil-sdk-ios-swift/Feed/FeedContainerViewController.swift
import BuzzvilSDK
import UIKit

// 하위 뷰 컨트롤러로 베네핏허브 연동하기
class FeedContainerViewController: UIViewController {
  let buzzAdFeed = BZVBuzzAdFeed { _ in }
  
  override func viewDidLoad() {
    super.viewDidLoad()
    displayContentViewController(buzzAdFeed.viewController)
  }
  
  private func displayContentViewController(_ contentViewController: UIViewController) {
    addChild(contentViewController)
    contentViewController.view.frame = view.bounds
    view.addSubview(contentViewController.view)
    contentViewController.didMove(toParent: self)
  }
}
